### ISSUE TITLE
hi 

### ISSUE THREAD

**Issue by hexiangquan** [kienvu58/carebro#1]
why cannot run your PRJ. many error

**Comment by kienvtv3** [comment]
Hi, I've just uploaded test videos and some changes.
You need keras with tensorflow backend, opencv3, python3 to run this code.
Try to run command: python main.py video\QuestionVideo.avi

**Comment by hexiangquan** [comment]
how about performance?

发自我的 iPhone

**Comment by kienvtv3** [comment]
High accuracy but cannot run realtime.

**Comment by hexiangquan** [comment]
it likes attention model one frame many times make it slow. but it can be parallel.make it faster


发自我的 iPhone

> 在 2017年5月7日，下午10:21，Kien Vu <notifications@github.com> 写道：
> 
> High accuracy but cannot run realtime.
>

**Comment by kienvtv3** [comment]
Yeah. We focused on accuracy and gived up speed. This project was a part of
a competition, but we were out so it was dropped.

**Comment by hexiangquan** [comment]
util now. .your team have find new method to find lane?  

发自我的 iPhone

> 在 2017年5月7日，下午10:31，Kien Vu <notifications@github.com> 写道：
> 
> Yeah. We focused on accuracy and gived up speed. This project was a part of
> a competition, but we were out so it was dropped.
>

**Comment by kienvtv3** [comment]
No. We didn't research about lane detection anymore.

**Comment by hexiangquan** [comment]
ok thank your reply your prj is awsome .it remind me a obstacle detection prj 
发自我的 iPhone

> 在 2017年5月7日，下午10:42，Kien Vu <notifications@github.com> 写道：
> 
> No. We didn't research about lane detection anymore.
>
